lib/organization.js

```diff
@@ -278,174 +269,174 @@ export class Organization {
-        options.params.schemeNativeTokenFee, // uint schemeNativeTokenFee; // a fee (in the present schemes token)  that is to be paid for submission
-        voteApproveParams, // bytes32 voteApproveParams;
-        votingMachine.address,
-      );
-
-      const feeToken = await scheme.nativeToken();
-      const fee = await scheme.fee();
-      const autoRegister = false;
-
-      tx = await schemeRegistrar.proposeScheme(
-        this.avatar.address, // Avatar _avatar,
-        scheme.address, //address _scheme,
-        parametersHash, // bytes32 _parametersHash,
-        false, // bool _isRegistering,
-        feeToken, // StandardToken _tokenFee,
-        fee, // uint _fee
-        autoRegister // bool _autoRegister
-      );
-      const proposalId = await getValueFromLogs(tx, '_proposalId');
-      return proposalId;
-    } else {
-      throw new Error('Unknown contract');
-    }
-  }
-
-  async proposeGlobalConstraint(opts= {}) {
-    const settings = await getSettings();
-    const defaults = {
-      contract: null,
-      address: null,
-      params: {},
-      paramsHash: null,
-      // next three options regard removing a global constraint
-      votingMachine: this.votingMachine.address,
-      reputation: this.reputation.address,
-      absPrecReq: 50,
-    };
-
-    const options = dopts(opts, defaults);
-
-    if (options.contract==='TokenCapGC') {
-      options.address = options.address || settings.daostackContracts.TokenCapGC.address;
-      const tokenCapGC = await TokenCapGC.at(options.address);
-
-      if (options.paramsHash) {
-          // TODO: check if paramsHash is registered
-      } else {
-          const defaultParams = {
-            token: null,
-            tokenAddress: this.token.address,
-            cap: 21e9,
-          };
-          let params = dopts(options.params, defaultParams);
-
-          await tokenCapGC.setParameters(params.tokenAddress, params.cap);
-          options.paramsHash = await tokenCapGC.getParametersHash(params.tokenAddress, params.cap);
-      }
-
-    } else {
-      if (options.address) {
-        //
-      } else {
-        let msg = 'Either "contract" or "address" must be provided';
-        throw new Error(msg);
-      }
-    }
-    // calculate (and set) the hash that will be used to remove the parameters
-    await AbsoluteVote.at(options.votingMachine).setParameters(options.reputation, options.absPrecReq, true);
-    options.votingMachineHash = await AbsoluteVote.at(options.votingMachine).getParametersHash(options.reputation, options.absPrecReq, true);
-
-    const globalConstraintRegistrar = await this.scheme('GlobalConstraintRegistrar');
-    let tx = await globalConstraintRegistrar.proposeGlobalConstraint(this.avatar.address, options.address, options.paramsHash, options.votingMachineHash);
-    const proposalId = getValueFromLogs(tx, '_proposalId');
-    return proposalId;
-  }
+  // async proposeScheme(opts={}) {
```

Comment:
why is all this commented out for now?

Reply:
I should have deleted those comments.  I plan to do that.